### DISCUSSION TITLE
🤔🤔🤔

### DISCUSSION THREAD

**Discussion by fearocanity** [fearocanity/ebtrfio-bot#9] [General]
めっちゃ投稿が速すぎない？...数日休んだ方がいいと思うわ。最近学校の課題に追われてるし、試験も近いし。それとも、このbotが勝手に回ってるの見守るだけにしとく？

**Comment by Rendevior** [comment]
> めっちゃ投稿が速すぎない？...数日休んだ方がいいと思うわ。最近学校の課題に追われてるし、試験も近いし。それとも、このbotが勝手に回ってるの見守るだけにしとく？

大丈夫、俺まだ時間を管理できるから、気にしなくていいよ(´∀`)b

**Reply by fearocanity** [reply]
> > めっちゃ投稿が速すぎない？...数日休んだ方がいいと思うわ。最近学校の課題に追われてるし、試験も近いし。それとも、このbotが勝手に回ってるの見守るだけにしとく？
> 
> 大丈夫、俺まだ時間を管理できるから、気にしなくていいよ(´∀`)b

じゃあ、任せるね。。

**Comment by fearocanity** [comment]
クラブ活動やパフォーマンス課題でまた忙しくなりそうだから、やばい感じになっちゃうかもしんないね。

**Reply by Rendevior** [reply]
俺もGitHubの時間ないわ、代わりに学業に集中するためにだらけてるんだよね。。

**Reply by Rendevior** [reply]
オフにしたほうがいいか？それとも最後のフレームまで走らせちゃう？めっちゃ迷うやんけ、どうすっかなー？

**Reply by fearocanity** [reply]
やっぱオフにした方がええわ、ポスティングでトラブル起こしたらアカンやろ。

**Reply by Rendevior** [reply]
> やっぱオフにした方がええわ、ポスティングでトラブル起こしたらアカンやろ。

じゃ、そうしよう、、

**Reply by fearocanity** [reply]
april fools day!

**Comment by fearocanity** [comment]
@Rendevior え、何があった？

**Reply by Rendevior** [reply]
Violation
